android/src/main/kotlin/live/hms/hmssdk_flutter/HmssdkFlutterPlugin.kt

```diff
@@ -95,6 +105,9 @@ class HmssdkFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
             "switch_video" -> {
                 switchVideo(call, result)
             }
+            "is_screen_share_active" -> {
```

Comment:
all screenshare related cases should be together, below one another
`is_screen_share_active` case should be around `start_screen_share` & `stop_screen_share`